src/components/Home.css

```diff
@@ -0,0 +1,32 @@
+.home_container {
```

Comment:
[OPTIONAL] It would be nicer to change the text color (yellow) or background color (white) of your website to improve the color contrast of your design:
- The page should have a minimum contrast ratio of at least 4.5:1 for normal-size text.
- You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.

Reply:
Hello @medaminedev66 
thanks for the time you spent reviewing my work, i will apply your valuable advises.